packages/web/src/lib/wallectconnect-mode.ts

```diff
@@ -39,6 +39,16 @@ export class WallectConnectSession {
     this.wcAccounts = this.connector.accounts;
   }
 
+  /**
+   * Kills a session if already exists, create new session
+   */
+  async forceCreate (): Promise<void> {
```

Comment:
how about adding a `force` parameter to the function below? By default it will be `false`

Reply:
yes

Reply:
the idea was to remove this function.

Reply:
ah yes, i forgot to remove it.